dependencies.properties

```diff
@@ -1,5 +1,5 @@
 version=5.4.0-SNAPSHOT
-amf=5.2.2-RC.0
+amf=5.2.2
 amf.custom-validator.js=1.2.3
-amf.custom-validator-scalajs=0.3.6-RC.2
+amf.custom-validator-scalajs=0.3.6
 amf-antlr-parsers=0.5.18
```

Comment:
Lean, does this version need to be bumped? I think the last release of amf-antlr is 0.6.21

Reply:
good catch